src/graph.rs

```diff
@@ -1846,7 +1803,10 @@ impl PyGraph {
 
     fn __delitem__(&mut self, idx: usize) -> PyResult<()> {
         match self.graph.remove_node(NodeIndex::new(idx)) {
-            Some(_) => Ok(()),
+            Some(_) => {
+                self.node_removed = true;
+                Ok(())
+            }
```

Comment:
This seems pretty obvious. Was there some reason this wasn't done previously?

Reply:
It was an oversight that's the second fix around hole handling (and what I added the release note for)